style: change margin value for the documentaion button in PolicyEdito…

Mohammed-Yasin-Mulla opened the pull request:
## I have read the [CONTRIBUTING.md](https://github.com/supabase/supabase/blob/master/CONTRIBUTING.md) file.

-  [x] YES

## What kind of change does this PR introduce?

This is a tiny style change to the modal. The documentation button is overlapping with the close button 

## What is the current behavior?

![image](https://github.com/user-attachments/assets/e356c7fc-4056-4bbe-a549-f7f015ceafb7)

## What is the new behavior?

![image](https://github.com/user-attachments/assets/b0eb88b4-00f0-45d5-b8e9-9b5dbaf331aa)


#28226

Mohammed-Yasin-Mulla committed:
style: change margin value for the documentaion button in PolicyEditorModal

saltcod commented:
Thanks @Mohammed-Yasin-Mulla !

Prob makes sense to solve this with padding on the container 
![screenshot-2024-07-29-at-21 50 00](https://github.com/user-attachments/assets/fc4b80ae-dc66-47c7-af52-07bd99910778)

Mohammed-Yasin-Mulla commented:
@saltcod ,
I have reverted the changes and added the `padding` as you mentioned, but I kept the `mt-[-4px]` to make all the elements have the same alignment.

I would love to know if there would be a better solution for `mt-[-4px]` on the button.
![image](https://github.com/user-attachments/assets/791cfa46-23d3-49f5-8bf6-ec1b79f2f622)

Mohammed-Yasin-Mulla commented:
@saltcod ,
I wanted to know if there are any other changes I need to add?
I hope the PR is good to merge.

Mohammed-Yasin-Mulla commented:
@saltcod , could you please approve the PR if its all good to go

fsansalvadore commented on apps/studio/components/interfaces/Auth/Policies/PolicyEditorModal/PolicyEditorModalTitle.tsx:
```diff
@@ -61,7 +61,7 @@ const PolicyEditorModalTitle = ({
           </Button>
         )}
-        <Button asChild type="default" icon={<IconExternalLink size={14} />}>
+        <Button asChild type="default" icon={<IconExternalLink size={14} />} className='mt-[-4px]' >
```
```suggestion
        <Button asChild type="default" icon={<IconExternalLink size={14} />} className='mt-[-4px]'>
```

fsansalvadore replied:
Prettier is failing, removing that final space should fix it @Mohammed-Yasin-Mulla

Mohammed-Yasin-Mulla replied:
thank you

Mohammed-Yasin-Mulla replied:
@fabianmossberg, did the refactoring of the code

saltcod committed:
Replace 3.5 with 4o

saltcod committed:
Merge branch 'master' of github.com:supabase/supabase

Hallidayo commented:
Hi @Mohammed-Yasin-Mulla - Got a conflict on this, once sorted should be all good to go 👍

saltcod committed:
Merge branch 'master' of github.com:supabase/supabase

saltcod committed:
Merge branch 'master' of github.com:supabase/supabase

saltcod committed:
Fix conflict

saltcod commented:
hey @Mohammed-Yasin-Mulla I accidentally made a mess of your branch, apologies. 

For simplicity, I've created a new brand and cherry-picked your good commit here
https://github.com/supabase/supabase/pull/29229/files

Mohammed-Yasin-Mulla commented:
Thank you @saltcod  👍🏼
